pkg/apis/extensions/v1alpha1/types.go

```diff
@@ -39,6 +40,8 @@ type Status interface {
 	GetLastError() *gardencorev1beta1.LastError
 	// GetState retrieves the State of the extension
 	GetState() *runtime.RawExtension
+	// GetResources retrieves the list of named resource references referred to in the state by their names.
+	GetResources() []gardencorev1alpha1.NamedResourceReference
```

Comment:
```suggestion
	GetResources() []gardencorev1beta1.NamedResourceReference
```

Reply:
I will rework as suggested. Just a question: since there is `NamedResourceReference` in both `v1alpha1` and `v1beta1`, what's the reason to prefer beta over alpha in a version that is itself alpha? Is it simply that it's the latest / more stable one, or is there something more?

Reply:
Yes, right, `v1beta1` is the latest + default version used by Gardener, that's the only reason.

Reply:
Thanks, done.